extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/AstroHandlerFactory.java

```diff
@@ -21,6 +20,7 @@
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
 
+import com.google.common.collect.Maps;
```

Comment:
Apart from these the concrete Thing Handlers (Sun and Moon) are using `com.google.common.collect` from its previous release.